### PR TITLE
Add error description to output

### PR DESCRIPTION
**Pull request by GokceGK**

### CODE REVIEW

File: internal/pkg/auth/user_login.go [hunk 1]
```diff
@@ -135,8 +135,9 @@ func AuthorizeUser(p *print.Printer, isReauthentication bool) error {
 
 		// Get the authorization code
 		code := r.URL.Query().Get("code")
+		errDescription := r.URL.Query().Get("error_description")
 		if code == "" {
-			errServer = fmt.Errorf("could not find 'code' URL parameter")
+			errServer = fmt.Errorf("could not find 'code' URL parameter.\nError: %s", errDescription)
```
Comment (vicentepinto98) [inline internal/pkg/auth/user_login.go:140]:
```suggestion
			errServer = fmt.Errorf("could not find 'code' URL parameter: %s", errDescription)
```

And probably also check and only concatenate `errDescription` if it is not empty

Comment (GokceGK) [reply internal/pkg/auth/user_login.go]: Added the check

File: internal/pkg/auth/user_login.go [hunk 2]
```diff
@@ -135,8 +135,12 @@ func AuthorizeUser(p *print.Printer, isReauthentication bool) error {
 
 		// Get the authorization code
 		code := r.URL.Query().Get("code")
+		errDescription := r.URL.Query().Get("error_description")
 		if code == "" {
 			errServer = fmt.Errorf("could not find 'code' URL parameter")
+			if errDescription != "" {
+				errServer = fmt.Errorf("%w.\nError: %s", errServer, errDescription)
```
Comment (vicentepinto98) [inline internal/pkg/auth/user_login.go:142]:
```suggestion
				errServer = fmt.Errorf("%w: %s", errServer, errDescription)
```

Comment (vicentepinto98) [reply internal/pkg/auth/user_login.go:142]: better to not introduce a paragraph and also "Error" is redundant

**Review by vicentepinto98 — APPROVED** [review]

**Pull request merged**